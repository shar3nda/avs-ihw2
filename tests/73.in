)A;Pi5-0AS1bok$r,F9GH}NjS{_2EI/T~7*ug.DpN%<*v_fb;)AJ1rR5K	cB$o<sm	Kn|\0L