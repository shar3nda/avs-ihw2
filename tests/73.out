Lowercase: 979, uppercase: 971